### ISSUE TITLE
Is there just one button?

### ISSUE THREAD

**Issue by 0208zhang** [emdanish75/Bookstore-App-In-WPF#1]

**Comment by emdanish75** [comment]
No, this is just a demo interface. We are in the process of making this app. It will be completed in 2 weeks and we'll commit the final project in this same repo.

Thanks for noticing though ;)
